ScoutSuite/providers/gcp/configs/services.py

```diff
@@ -31,7 +36,7 @@ def __init__(self, metadata=None, thread_config=4, projects=None, **kwargs):
         except NameError as e:
             pass
 
-        self.stackdriverlogging = StackdriverLoggingConfig(thread_config=thread_config)
+        self.stackdriverlogging = StackdriverLogging(gcp_facade, stackdriverlogging_facade)
```

Comment:
Why do you have two facades? It kind of goes against the purpose of having a facade. I suggest including the StackDriver facade in the GCP facade (and use it like so: `gcp_facade.stackdriver.get_sinks(...)`)

Reply:
Done!